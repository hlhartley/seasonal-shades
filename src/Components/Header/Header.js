import React from 'react';

export const Header = () => {
    return(
        <div className='header-container'>
            {/* <div className='search-container'>
                <input className='search-input'></input>
                <i class="fas fa-search"></i>
            </div> */}
            <h1 className='title'>Seasonal Shades</h1>
            <div>
                <i className="fas fa-user-circle"></i>
                <i className="fas fa-heart"></i>
            </div>
        </div>
    )
};

