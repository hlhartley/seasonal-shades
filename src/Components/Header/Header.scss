.header-container {
    align-items: center;
    display: flex;
    justify-content: space-between;
    
    .title {
        font-family: $title-font;
        font-size: 55px;
        letter-spacing: 3.5px;
        margin-left: 25px;
    };
    
    .fas {
        font-size: 25px;
        margin: 0 15px;
    };

    .fa-heart {
        margin-right: 25px;
    };
}

// .search-container {
//     display: flex;
//     align-items: center;

//     .search-input {
//         height: 25px;
//         margin-left: 25px;
//     };

//     .fa-search {
//         font-size: 24px;
//     };
// }
