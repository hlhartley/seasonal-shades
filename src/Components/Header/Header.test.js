import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Header />)
    });

    it('should match the snapshot with all the data passed in correctly', () => {
        expect(wrapper).toMatchSnapshot()
    });
});
