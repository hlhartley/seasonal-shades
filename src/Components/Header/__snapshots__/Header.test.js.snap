// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match the snapshot with all the data passed in correctly 1`] = `
<div
  className="header-container"
>
  <h1
    className="title"
  >
    Seasonal Shades
  </h1>
  <div
    className="user-info"
  >
    <p>
      Welcome, User!
    </p>
    <NavLink
      activeClassName="active"
      aria-current="page"
      to="/favorites"
    >
      <i
        className="fas fa-heart"
        data-for="tooltip"
        data-tip={true}
      />
    </NavLink>
    <ReactTooltip
      effect="solid"
      id="tooltip"
      insecure={true}
      resizeHide={true}
      type="dark"
      wrapper="div"
    >
      View favorites
    </ReactTooltip>
  </div>
</div>
`;
